Formula/awscli.rb

```diff
@@ -135,7 +140,7 @@ def caveats
 
   test do
     assert_match "topics", shell_output("#{bin}/aws help")
-    assert_includes Dir[libexec/"lib/python3.9/site-packages/awscli/data/*"],
-                    "#{libexec}/lib/python3.9/site-packages/awscli/data/ac.index"
+    site_packages = libexec/Language::Python.site_packages("python3")
```

Comment:
Test doesn't set up Python dependencies as part of environment (PATH/etc), so it will use system Python 3.8.

So, still need to hardcode version number.